src/browser.rs

```diff
@@ -604,8 +604,13 @@ impl BrowserConfigBuilder {
         self
     }
 
-    pub fn viewport(mut self, viewport: Viewport) -> Self {
-        self.viewport = viewport;
+    /// Configures the viewport of the browser, which defaults to `800x600`.
+    /// `None` disables viewport emulation (i.e., it uses the browsers default
+    /// configuration, which fills the available space. This is similar to what
+    /// Playwright does when you provide `null` as the value of its `viewport`
+    /// option).
+    pub fn viewport(mut self, viewport: impl Into<Option<Viewport>>) -> Self {
```

Comment:
impl into Option makes it harder to use `None`
```suggestion
    pub fn viewport(mut self, viewport: Option<Viewport>) -> Self {
```

Reply:
@mattsse: I don't care much if we change that part, but I'm not sure what you mean with "impl into Option makes it harder to use `None`".

To use `None`, you'd just use `config.viewport(None)`.

Reply:
> To use None, you'd just use config.viewport(None).

try it